ega-data-api-res/src/main/java/eu/elixir/ega/ebi/reencryptionmvc/exception/GeneralStreamingException.java

```diff
@@ -26,6 +26,10 @@
 
     private static final long serialVersionUID = 1L;
 
+    public GeneralStreamingException(String error) {
+        super(error);
+    }
+    
     public GeneralStreamingException(String error, int stage) {
         super("Error processing Stream at stage : " + stage + " (with error " + error + ")");
```

Comment:
same as above - log the exception with a name that we can see clearly in Kibana what is going on - and filter by it.

Reply:
@afoix Could you pls elaborate more on how do you want it?

Reply:
Yes, all the exceptions are logging like this in Kibana, so you really don't know what they are about.